torchtune/training/quantization.py

```diff
@@ -74,6 +82,45 @@ def quantize(self, model):
 ] = enable_8da4w_fake_quant_module_swap
 
 
+class Int8MixedPrecisionTrainingQuantizer:
+    """Apply INT8 mixed-precision training. During training, weights and activations
```

Comment:
Great docstring. Since its a heavy topic, would you mind adding:

1) The expected benefits and tradeoffs? E.g. the user may be confused if this is for memory and if it impacts loss.
2) Args (argtype): arg comment
3) Alink to learn more about int8 mixed precision, if there is one?